sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveCatalogs.scala

```diff
@@ -98,3 +83,20 @@ class ResolveCatalogs(val catalogManager: CatalogManager)
     }
   }
 }
+
+object ResolveCatalogs {
+  def resolveNamespace(
```

Comment:
Extract method `resolveNamespace` from `class ResolveCatalogs` to `object ResolveCatalogs` and reuse it in `UnsetNamespacePropertiesCommand`